skaffold/src/main/resources/messages/SkaffoldBundle.properties

```diff
@@ -20,7 +20,7 @@ skaffold.run.config.general.description=Runs Skaffold Deployment
 skaffold.run.config.single.run.name=Deploy to Kubernetes with Skaffold
 skaffold.run.config.single.run.helperText=Uses Skaffold to build and deploy your application to Kubernetes cluster exactly once.
 skaffold.run.config.dev.run.name=Continuous development on Kubernetes with Skaffold
-skaffold.run.config.dev.run.helperText=Monitors the source code, so that on any change, IDE builds and deploys your application to Kubernetes cluster.
+skaffold.run.config.dev.run.helperText=Monitors the project, so that on any change, Skaffold builds and deploys your application to Kubernetes cluster.
```

Comment:
lgtm regarding the IDE -> Skaffold change.

For "monitors the project'. I wonder if we can come up with simple wording that is even more transparent. Something about monitoring dependencies in the Docker file.

For reference. the Skaffold project says:

> Watches your source code and the dependencies of your docker images for changes and runs a build and deploy when changes are detected

Although the first part about "watching your source code" isn't necessarily true.

Reply:
`Monitors the dependencies of your docker images, so that ...`?

Reply:
`Watches the dependencies of your docker image for changes, so that ` ?